core/spec/models/spree/price_spec.rb

```diff
@@ -14,11 +14,6 @@
     let(:variant) { stub_model Spree::Variant }
     subject { Spree::Price.new variant: variant, amount: amount }
 
-    context 'when the amount is nil' do
```

Comment:
Why not keeping this spec and invert the expectation instead?

Reply:
Fair enough... Done!